manifests/charts/base/templates/clusterrole.yaml

```diff
@@ -9,7 +9,7 @@ rules:
   # sidecar injection controller
   - apiGroups: ["admissionregistration.k8s.io"]
     resources: ["mutatingwebhookconfigurations"]
-    verbs: ["get", "list", "watch", "patch"]
+    verbs: ["get", "list", "watch", "update", "patch"]
```

Comment:
It seems like you're removing the ability to patch and changing it to update below. Is patching still supported?

Reply:
There is no code that will patch but I think the clusterrole is shared between instances so it would break upgrades to remove the patch permission

Reply:
Ahh, makes sense. Thanks.